index.php

```diff
@@ -1,15 +1,3 @@
 <?php
-
-define('ROOT_DIR', realpath(dirname(__FILE__)) . '/');
```

Comment:
What is the purpose of moving those constants in a separat file ? Why not moving them to the Pico class constructor for example ?